### PR TITLE
Add new property `log4j2XML`

### PR DESCRIPTION
**Pull request by sebglon**
Fix issue: https://github.com/pmint93/helm-charts/issues/52

### CODE REVIEW

File: charts/metabase/templates/deployment.yaml [hunk 1]
```diff
@@ -194,6 +198,10 @@ spec:
             - name: config
               mountPath: /tmp/conf/
             {{- end }}
+            {{- if .Values.log4j2XML }}
```
Comment (ClementGautier) [inline charts/metabase/templates/deployment.yaml:201]: You will have a conflict if you pass both log4j2XML and log4jProperties. I suggest you simply modify the condition on line 197 like this: `{{- if .Values.log4jProperties or .Values.log4j2XML }}`

File: charts/metabase/templates/deployment.yaml [hunk 2]
```diff
@@ -264,6 +272,14 @@ spec:
             - key: log4j.properties
               path: log4j.properties
         {{- end }}
+        {{- if .Values.log4j2XML}}
```
Comment (ClementGautier) [inline charts/metabase/templates/deployment.yaml:275]: Same comment, this time you can probably get around with something like this:
```
        {{- if .Values.log4jProperties or .Values.log4j2XML }}
        - name: config
          configMap:
            name: {{ template "metabase.fullname" . }}-config
            items:
            {{- if .Values.log4jProperties}}
            - key: log4j.properties
              path: log4j.properties
            {{- end }}
            {{- if .Values.log4j2XML}}
            - key: log4j2.xml
              path: log4j2.xml
            {{- end }}
        {{- end }}
```

**Review by ClementGautier — CHANGES_REQUESTED** [review]

File: charts/metabase/templates/deployment.yaml (hunk 1 above)
Comment (sebglon) [reply charts/metabase/templates/deployment.yaml]: Fixed

File: charts/metabase/templates/deployment.yaml (hunk 2 above)
Comment (sebglon) [reply charts/metabase/templates/deployment.yaml]: Fixed

**Review by ClementGautier — APPROVED** [review]

**Review by pmint93 — APPROVED** [review]

**Pull request merged**